web-app/src/app/stitch-upload.html

```diff
@@ -36,16 +36,16 @@
           uploadUrl: {
             type: String,
             value: `${SM.apiUrl}/api/patterns`
-          },
-          authHeader: {
-            type: String,
-            value: ''
           }
         }
       }
 
-      getAuthToken() {
-        this.authHeader = JSON.stringify(window.getAuthHeaders());
+      connectedCallback() {
+        super.connectedCallback();
+        const upload = this.$.upload;
+        upload.addEventListener('upload-before', function (event) {
```

Comment:
I would rather use arrow function and accessed `this.$.upload` instead of defining variable.